addon/index.js

```diff
@@ -27,6 +27,9 @@ export {
 if (typeof define !== 'undefined' && define.amd){
   define('ember-inflector', ['exports'], function(__exports__){
     __exports__['default'] = Inflector;
+    __exports__.pluralize = pluralize;
+    __exports__.singularize = singularize;
+
     return Inflector;
   });
 } else if (typeof module !== 'undefined' && module['exports']){
```

Comment:
When this check is true, we also need to export singularize/pluralize.

Probably better to do:
```javascript
module['exports'] = {
  'Inflector': Inflector,
  'singularize': singularize,
  'pluralize': pluralize
}
```

thoughts?